http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/PackageSpec.scala

```diff
@@ -16,4 +16,11 @@ class PackageSpec extends AnyFlatSpec with Matchers {
     val duplicateLabel = Map("_labels" -> "foo=bar,foo=biz")
     processLabelMap(duplicateLabel) shouldBe (Right(Map("foo" -> "biz")))
   }
+
+  it should "process labels to return properly" in {
+    val input1 = Map("includeLabels" -> "foo,bar")
+    processLabelsToReturn(input1, None) shouldBe Right(List("foo", "bar"))
+
+    // TODO: what would a bad request look like?
```

Comment:
one example, val input1 = Map("includeLabels" -> "foo;bar")

Reply:
that would process as `["foo;bar"]` and not fail - i think i need more logic in `processLabelsToReturn` so that we catch that kind of exception. what else would be considered an exception? happy to jump on a call for this if it's easier

Reply:
it's probably better to use regex instead of just simple `split` then

Reply:
i was thinking to use regex as well - i'll update